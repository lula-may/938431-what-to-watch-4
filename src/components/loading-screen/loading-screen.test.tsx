import * as React from "react";
import * as renderer from "react-test-renderer";
import LoadingScreen from "./loading-screen";

describe(`LoadingScreen Component`, () => {
  it(`should render correctly LoadingScreen component`, () => {
    const tree = renderer.create(
        <LoadingScreen/>
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
