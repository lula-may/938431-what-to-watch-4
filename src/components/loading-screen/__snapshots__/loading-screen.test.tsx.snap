// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen Component should render correctly LoadingScreen component 1`] = `
Array [
  <section
    className="movie-card"
  >
    <h1
      className="visually-hidden"
    >
      WTW
    </h1>
    <header
      className="page-header movie-card__head"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
    </header>
    <h3
      style={
        Object {
          "marginBottom": "160px",
          "marginTop": "180px",
          "textAlign": "center",
        }
      }
    >
      Loading...
    </h3>
  </section>,
  <div
    className="page-content"
  >
    <footer
      className="page-footer"
    >
      <div
        className="logo"
      >
        <a
          className="logo__link logo__link--light"
        >
          <span
            className="logo__letter logo__letter--1"
          >
            W
          </span>
          <span
            className="logo__letter logo__letter--2"
          >
            T
          </span>
          <span
            className="logo__letter logo__letter--3"
          >
            W
          </span>
        </a>
      </div>
      <div
        className="copyright"
      >
        <p>
          Â© 2019 What to watch Ltd.
        </p>
      </div>
    </footer>
  </div>,
]
`;
